docs/pages/experiments/md3/index.tsx

```diff
@@ -280,14 +280,17 @@ const customPalette = {
   neutral: {
     '0': '#000000',
     '10': '#201a1c',
+    '17': '#2e282a',
```

Comment:
We will need to update the types too. Are these values coming from the specs?

Reply:
Updated them here 😊: https://github.com/mui/material-ui/blob/c59f9233af168af0cb5ada04eedf2d43c7c31c4c/packages/mui-material-next/src/styles/Theme.types.ts#L28

Yes, they come from the specs. Here are [the values for the MD3 default theme](https://github.com/material-components/material-web/blob/main/tokens/v0_172/_md-ref-palette.scss#L46), and here I added them to our palette:

https://github.com/mui/material-ui/blob/c59f9233af168af0cb5ada04eedf2d43c7c31c4c/packages/mui-material-next/src/styles/palette.ts#L81

For this experiment file, I noticed that it didn't use the default MD3 colors so I created the tones based on the existing neutral tones, altering the light value as explained in: https://m3.material.io/styles/color/the-color-system/key-colors-tones#a828e350-1551-45e5-8430-eb643e6a7713